src/Chapter2.hs

```diff
@@ -336,7 +347,9 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList n m list
+  | n < 0 = []
```

Comment:
Your implementation is correct when the right bound is lower than the left. But I think you can implement it more efficiently by adding a separate case to your check :slightly_smiling_face:

Reply:
not too sure if I get you correctly, you are looking for another case where `m < n` as an additional case to return empty list?

Reply:
Yeah, I meant exactly that 🙂